build.gradle

```diff
@@ -30,6 +30,7 @@ dependencies {
 	annotationProcessor 'org.projectlombok:lombok'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 	testImplementation 'org.springframework.security:spring-security-test'
+	testImplementation 'org.hamcrest:hamcrest:2.1'
```

Comment:
is it a part of starter test?

Reply:
This dependency necessary for assertThat assertion

Reply:
I mean this dependency is part of org.springframework.boot:spring-boot-starter-test

Reply:
Ok, got it